tests/features/bpmsuite/smartcontroller/smartcontroller.feature

```diff
@@ -0,0 +1,15 @@
+@jboss-bpmsuite-7/bpmsuite70-standalonecontroller-openshift
```

Comment:
Should we call the file `standalonecontroller.feature` instead of `smartcontroller.feature`?

Reply:
yes, thanks.